### PR TITLE
docs(journey-maps): updated `journey-maps` maplibre css version to 2.4.0

### PR DESCRIPTION
**Pull request by mstankala**

### CODE REVIEW

**Review by swiss-chris — APPROVED** [review]
LGTM

File: src/journey-maps/angular/angular.md [hunk 1]
```diff
@@ -15,12 +15,12 @@ npm install --save @sbb-esta/journey-maps
 ```
 
 ```sh
-npm install --save maplibre-gl@2.1.9
+npm install --save maplibre-gl@2.4.0
```
Comment (sebisteiner) [inline src/journey-maps/angular/angular.md:18]:
```suggestion
npm install --save maplibre-gl
```

File: src/journey-maps/angular/angular.md [hunk 2]
```diff
@@ -15,12 +15,12 @@ npm install --save @sbb-esta/journey-maps
...
 npm install --save-dev @types/geojson
 ```
 
 **NOTE** \
-The `maplibre-gl` version compatible with this version of `journey-maps` is `2.1`.
+The `maplibre-gl` version compatible with this version of `journey-maps` is `2.4`.
```
Comment (sebisteiner) [inline src/journey-maps/angular/angular.md:23]:
```suggestion
The `maplibre-gl` version compatible with this version of `journey-maps` is `2.x.y`.
```

**Review by sebisteiner — CHANGES_REQUESTED** [review]

**Review by sebisteiner — APPROVED** [review]
LGTM 👍

**Pull request merged**